metals/src/main/scala/scala/meta/internal/metals/Messages.scala

```diff
@@ -382,4 +382,10 @@ class Messages(icons: Icons) {
         s"Please upgrade to Scala $recommended."
     }
   }
+
+  object NewScalaFile {
+    def selectTheKindOfFileMessage = "Select the kind of file to create"
+    def enterNameMessage(kind: String): String =
+      s"Enter the name for the new $kind"
```

Comment:
Super nitpicky, but I just tested this, and is it possible to just add a `:` in here?
```suggestion
      s"Enter the name for the new $kind:"
```

Reply:
it seems not to be going look good here:
![image](https://user-images.githubusercontent.com/10850363/73563467-adfe6880-445d-11ea-89f6-a9a6398034da.gif)

Reply:
Let's leave it as is, I don't think it adds that much visually and it's not really being used in UI anywhere.

Reply:
Hmmm, yea I see what you're saying. That's tricky, because in `coc-metals` there isn't a nice break, so it seems like there should really be a `:`. No worries, I'll just add it in on the vim side.

<img width="438" alt="Screenshot 2020-02-24 at 14 01 42" src="https://user-images.githubusercontent.com/13974112/75154558-3b0ba900-570e-11ea-81fa-d6ef9b2fbfd5.png">

Reply:
Also, this feature works great 🎉 !!

Reply:
thanks :relieved: